user/views.py

```diff
@@ -1,5 +1,4 @@
 from rest_framework import generics
-from rest_framework.authentication import TokenAuthentication
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.authtoken.views import ObtainAuthToken
 from rest_framework.settings import api_settings
```

Comment:
remove ObtainAuthToken as it's used only in CreateTokenView that isn't used anymore. Also remove AuthTokenSerializer import and class itself